<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherManagement</class>
 <widget class="QDialog" name="TeacherManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Teachers</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#TeacherManagement {
        background-color: #fef6f0   ;
        border: 1px solid black;
        border-radius: 10%;
}</string>
  </property>
  <widget class="QTableWidget" name="teacherTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>781</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget#teacherTable {
    border: 1.5px solid #2c3e50;
    border-radius: 8px;
    background-color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
    font-size: 11pt;
    color: #2c3e50;
    gridline-color: #ecf0f1;
    alternate-background-color: #f7f9fa;
    selection-background-color: #d6eaf8;
    selection-color: #1a252f;
}

QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    font-weight: bold;
    font-size: 11pt;
    padding: 6px;
    border: none;
    font-family: 'Segoe UI', sans-serif;
}

QTableWidget QTableCornerButton::section {
    background-color: #2c3e50;
    border: none;
}

QTableWidget::item {
    padding: 6px;
    font-size: 10.5pt;
}

QTableWidget::item:selected {
    background-color: #d6eaf8;
    color: #1a252f;
}
</string>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <property name="horizontalHeaderLabels" stdset="0">
    <stringlist>
     <string>ID</string>
     <string>Name</string>
     <string>Subject</string>
    </stringlist>
   </property>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="labelSubject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelSubject {
        font-size: 20px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelName {
        font-size: 20px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>81</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#subjectLineEdit {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>31</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#nameLineEdit {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editTeacherButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#editTeacherButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
color: #2c3e50 ;
}

QPushButton#editTeacherButton:hover {
color: #2c3e50 ;
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>Edit Teacher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewTeacherButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>600</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#viewTeacherButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
color: #2c3e50 ;
}

QPushButton#viewTeacherButton:hover {
color: #2c3e50 ;
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>View Teacher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeTeacherButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#removeTeacherButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
color: #2c3e50 ;
}

QPushButton#removeTeacherButton:hover {

color: #2c3e50 ;
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>Remove Teacher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTeacherButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#addTeacherButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
color: #2c3e50 ;
}

QPushButton#addTeacherButton:hover {
color: #2c3e50 ;
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>Add Teacher</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
